src/Parts/Component.php

```diff
@@ -13,7 +13,7 @@ class Component
 {
     public MessageComponentTypes $type;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Component[]
+     * @var Component[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)